studio/components/interfaces/Home/ProjectList/ProjectList.tsx

```diff
@@ -40,7 +40,7 @@ const ProjectList = ({ rewriteHref }: ProjectListProps) => {
   const projectsByOrg = groupBy(allProjects, 'organization_id')
   const isLoadingPermissions = IS_PLATFORM ? _isLoadingPermissions : false
 
-  return (
+  return organizations && organizations?.length > 0 ? (
```

Comment:
Can we use `isSuccess` from the query here so we don't show `<NoProjectsState />` while it's loading? It might be okay because the loading state could be handled further up, but nice for consistency either way 😄